### PR TITLE
Update MSP clusterpool configuration labels to claim based on size and multi-az

### PR DESCRIPTION
**Pull request by samvarankashyap**
$subject

### CODE REVIEW

File: ci-operator/config/redhat-openshift-ecosystem/msp/redhat-openshift-ecosystem-msp-ocp-4.8.yaml [hunk 1]
```diff
@@ -19,6 +19,8 @@ tests:
     product: ocp
     timeout: 1h0m0s
     version: "4.8"
+    multi-az: "false"
```
Comment (dirgim) [inline ci-operator/config/redhat-openshift-ecosystem/msp/redhat-openshift-ecosystem-msp-ocp-4.8.yaml:22]: The new labels will also need to be added to the `cluster_claim` section of the  `small-single-az-amd64-aws` job in this same file.

Comment (samvarankashyap) [reply ci-operator/config/redhat-openshift-ecosystem/msp/redhat-openshift-ecosystem-msp-ocp-4.8.yaml:22]: checking

Comment (samvarankashyap) [reply ci-operator/config/redhat-openshift-ecosystem/msp/redhat-openshift-ecosystem-msp-ocp-4.8.yaml:22]: done @dirgim

**Pull request merged**